src/AutoRest.CSharp.V3/Program.cs

```diff
@@ -23,16 +22,17 @@ public static async Task<int> Main(string[] args)
             // Initialize workspace in the background
             GeneratedCodeWorkspace.Initialize();
 
-            if (args.Contains("--standalone"))
-            {
-                await StandaloneGeneratorRunner.RunAsync(args);
-                return 0;
-            }
-
-            if (args.Contains("--launch-debugger") && !Debugger.IsAttached)
-            {
-                Debugger.Launch();
-            }
+            //if (args.Contains("--standalone"))
+            //{
+            //    await StandaloneGeneratorRunner.RunAsync(args);
+            //    return 0;
+            //}
+
+            //if (args.Contains("--launch-debugger") && !Debugger.IsAttached)
+            //{
+            //Debugger.Launch();
+            //}
+            await Task.CompletedTask;
```

Comment:
Ignore this. I did this to circumvent the normal running stuff since `--launch-debugger` wasn't working. Something is weird with the PS scripts.